_includes/datagov-search-display.html

```diff
@@ -4,17 +4,46 @@
       <div class="row">
           <div id="database-search-container" class="col">
           <div class="field has-addons">
-              <div class="control has-icons-left is-expanded">
-                  <input class="input is-fullwidth is-large has-border-grey-light" id="search-box-datagovsg" type="text" placeholder="What are you looking for?" name="query" autocomplete="off">
+              <div class="control has-icons-left is-expanded datagov-search-border is-flex is-vh-centered">
+                  <input class="input is-fullwidth is-large has-border-grey-light is-size-5 pl-12" style="height:3.25rem" id="search-box-datagovsg" type="text" placeholder="What are you looking for?" name="query" autocomplete="off">
                   <span class="is-large is-left">
-                      <i class="sgds-icon sgds-icon-search is-size-4 search-bar"></i>
+                      <i class="sgds-icon sgds-icon-search is-size-5 search-bar"></i>
                   </span>
 
               </div>
-              <div class="control">
+              <span class="px-4 is-hidden-touch is-size-5" style="align-self:center;">
+                in
+              </span>
```

Comment:
Can I confirm that this "in" will not appear for smaller viewports?

Reply:
yup, can view in the figma as well! It only shows up in the desktop view